source/app-sharing.rst

```diff
@@ -16,61 +18,67 @@ Overview
    home directory that other users can launch. This is similar to adding to your
    PATH the bin directory of another user.
 
-The rest of this document addresses OnDemand's support for Executable Sharing.
+File System Permissions
+-----------------------
 
-.. warning:: Executable sharing means the app and all its code runs as the user
-             executing it, like everything else in OnDemand. User's might not
-             realize this. We currently do not provide an opt in screen warning
-             users that this app "will have permission to do everything on their
-             behalf and act as them". As a result, you should fully trust whoever
-             you enable to do share apps using executable sharing.
+Admins may install apps to the system by copying the application directory to ``/var/www/ood/apps/sys``. Default directory permissions (``755``) will allow all users with access to Ondemand to see and run that app. Apps may have their access restricted by changing the permissions on individual application directories. For example if a site does not wish to show licensed software to un-licensed users they might do the following:
 
+  .. code-block:: sh
 
-Demo
-----
+    # Given:
+    # - an app named $NEW_APP
+    # - a group named $NEW_APP_GROUP
+    # - and a user named $NEW_APP_USER
 
-This is with two users Eric (efranz) and Bob (an0047).
+    sudo cp "/path/to/$NEW_APP" /var/www/ood/apps/sys
+    sudo chmod 750 "/var/www/ood/apps/sys/$NEW_APP"
+    sudo chgrp "$NEW_APP_GROUP" "/var/www/ood/apps/sys/$NEW_APP"
 
-Eric has a dev app "Matlab", and interactive plugin app. Eric can
+    sudo usermod -a -G "$NEW_APP_GROUP" "$NEW_APP_USER"
 
-1. Launch Matlab
-2. View and Edit the code
+Using file access lists (FACLs) app sharing may be as specific as desired; sharing with one or more groups, potentially also including or excluding individual users.
 
-.. figure:: /images/app-sharing-1.png
-   :align: center
+Another use for app sharing is sharing an in-development application with members of a development team before deploying that app to the system.
```

Comment:
I would remove this sentence.

Reply:
Why? It's my primary use for app sharing.

Reply:
This sentence is in a section on System installed apps and app authorization.

Reply:
If you want to keep it you can reword as follows:

>This app authorization mechanism (using file permissions) can also be useful for canary deployments (sharing an app under development with a subset of users, or just your development team).